### DISCUSSION TITLE
When someone modifies a filter, it should show what percentage of people they are eliminating by applying it.

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#129] [Ideas & Features]
For example, if someone sets their height requirement to over 6 feet, it should show something like "are you sure? This choice eliminates 75% of potential matches". This way people are at least aware of what their preferences are doing to their dating prospects.

**Comment by ajbott** [comment]
This is a top feature in letting them know on how their "non-negotiable" / filtering affects how many people actually meet their standards.
